logstreams/src/main/java/io/zeebe/logstreams/storage/atomix/AtomixLogStorageReader.java

```diff
@@ -141,6 +139,36 @@ private long readFirstBlock() {
     return Optional.empty();
   }
 
+  private long seekToFirst() {
+    // looking for ASQN 0 is a trick which will skip any initial records with ASQN_IGNORE
+    // but stop as soon as we find an ASQN with 0 or greater (and we typically start at 1)
+    return findEntry(0).map(Indexed::index).orElse(LogStorage.OP_RESULT_INVALID_ADDR);
+  }
+
+  /** @deprecated should be dropped and handled properly by the RaftLogReader or the journal */
+  @Deprecated(since = "1.0.0")
```

Comment:
Is it really necessary? It is a private method :grinning:

Reply:
I wanted it to be explicit so we would know to remove it in the future, or question why we're still using it :smile: